jso/apis/src/main/java/org/teavm/jso/dom/types/DOMTokenList.java

```diff
@@ -30,7 +30,17 @@ public interface DOMTokenList extends JSObject {
 
     void add(String token);
 
+    void add(String token1, String token2);
+
+    void add(String... tokens);
```

Comment:
Are you sure that varargs supported in JSO?

Reply:
There also was a comment to create issue to support varargs in JS. Ok, will create issue to support varagrs.

Reply:
So are they supported or not? If they aren't, I can't accept PR with these declarations

Reply:
I thought that you know as author of TeaVM. But ok, will check.

Reply:
I'm not a machine remebering each and every character from this repository

Reply:
Sorry, didn't want to offend you. Just checked. Varargs are not supported. So, extracted varargs to separate PR https://github.com/konsoletyper/teavm/pull/867